### ISSUE TITLE
MAX - Allow us to lower the SSAO value even further.

### ISSUE THREAD

**Issue by harkinhails** [TheGameCreators/GameGuruRepo#5326]
Current SSAO values are too high (for me at-least).

![04 SSAO](https://github.com/TheGameCreators/GameGuruRepo/assets/102403572/a0c3b1a4-d284-4f03-aa26-aab3e05f8cbd)

At 0.4, too high.

![001 SSAO](https://github.com/TheGameCreators/GameGuruRepo/assets/102403572/94d424fe-a2c4-444b-9f7d-a9bbcd8d87bf)

At 0.01, still too high. 

I want SSAO to be subtle, like I'm used to doing in Vray in 3dsMax. These current values just make things look too ugly.. and unrealistic.

**Comment by OldFlak** [comment]
Yes, ssao should be way more subtle. I would think 0.01 would be practically off?
